### DISCUSSION TITLE
Why is my telescope recognized as person😂

### DISCUSSION THREAD

**Discussion by t0ny-peng** [blakeblackshear/frigate#1266] [General]
It happened several times that the telescope is detected as a person. This usually happens when the light goes on, even though that area has been masked. 

I just want to understand the reason behind this. Also, is there a place where I can find the info about the model used for detection? SSD? YOLO? Thx

![image](https://user-images.githubusercontent.com/4816327/123042496-eab95f80-d3c4-11eb-80c4-6d0d17dfd58b.png)

![image](https://user-images.githubusercontent.com/4816327/123042781-53084100-d3c5-11eb-8a76-abb5a11268bf.png)

**Comment by ElMoribond** [comment]
It is not the telescope that is detected but the person lying on your sofa :upside_down_face:

**Comment by hagak** [comment]
I have a scope that is also often flagged as a person as well.  I ended masking it out.

**Reply by hagak** [reply]
Just read blakeblackshear in another post mention to only mask out items like time stamps, trees.  So my suggestion of masking out the scope is probably not the best one.  Might be better to make a zone for everything else and only alert on things entering that zone.

**Comment by blakeblackshear** [comment]
Frigate uses the SSD models provided by Google for the edgetpu. This will happen until I develop my own model. You can adjust filters to try and avoid the false positive. 

Adding a motion mask does not prevent objects from being detected. Please read the documentation on masks: https://blakeblackshear.github.io/frigate/configuration/cameras#masks
